CONTRIBUTING.md

```diff
@@ -85,6 +85,15 @@ NuGet members may contribute directly to the main remote.
 
     > Note: You have to to run .\configure.ps1 and .\build.ps1 at least once in order for your build to succeed.
 
+1. Run dotnet code formatters and correct any errors.
+    * You can use `Format Document` in VS: 
+
+       `Ctrl+K, Ctrl+D` or Edit > Advanced > Format Document (https://docs.microsoft.com/visualstudio/ide/default-keyboard-shortcuts-in-visual-studio?view=vs-2019#text-editor)
+
+    * You can use the dotnet CLI tool (https://github.com/dotnet/format):
+
+      `dotnet format --check --exclude submodules`
```

Comment:
`packages` and `cli` need to be excluded as well.

Reply:
fixed

Reply:
I don't believe it does need to exclude cli and packages. When running `dotnet format`, it clearly says it's checking the projects in `nuget.sln`, so it's not searching for all code files, it's searching for code files used by projects used by nuget.sln (meaning that the Apex projects and source code are not being checked).